ogc/features/templates/features.go.html

```diff
@@ -96,5 +127,52 @@ <h5 class="card-header">
     {{ end }}
     </div>
 
+    <div class="col-4" >
+        <table>
+            <tr>
+                <td class="w-auto px-2">
+                    <b>{{ i18n "Example" }} URL</b>
+                </td>
+            </tr>
+            <tr>
+                <td class="w-auto px-2">
+                    <code id="field-url-example"></code>
+                </td>
+            </tr>
+        </table>
+
+        <link rel="stylesheet" type="text/css" href="view-component/styles.css">
+        <script type="text/javascript" src="view-component/main.js"></script>
+        <script type="text/javascript" src="view-component/polyfills.js"></script>
+        <script type="text/javascript" src="view-component/runtime.js"></script>
+        <app-feature-view id="viewer" background-map="BRT" projection="EPSG:28992">
```

Comment:
default projectie staat nu op RD, dit moet WGS84 zijn. Je kan daarvoor `projection="EPSG:28992"` helemaal weglaten. En deze dus alleen instellen op het moment dat een andere CRS (met EPSG code) is geselecteerd.

cc @JLSchaap